spec/rubocop/cop/rspec/instance_variable_spec.rb

```diff
@@ -67,6 +67,41 @@ def serialize
     RUBY
   end
 
+  context 'when used in a custom matcher' do
+    it 'ignores instance variables inside `mather`' do
+      expect_no_offenses(<<~RUBY)
```

Comment:
for full coverage, maybe add an instance usage (@matcher) outside of the matcher definition?

Reply:
Sometimes I see you as a clairvoyant, Maxim, that spec immediately helped to find a bug. 👍